### PR TITLE
Fixed WebPi feed

### PR DESCRIPTION
**Pull request by shrimpy**
Fixed WebPi feed dependencies; 
Update gitignore file to ignore WebPi log.

### CODE REVIEW

**Comment by suwatch** [comment]
Looks good - I don't have a new VM to actually try it.   BTW, we have 3 envs in play.

- local machine
- kudu-ci
- antares (Kudu2 and Kudu3)

Ideally we would love to keep them the same - so what we test locally will highly likely work the same on Antares.   However, it seems hard given Antares moves at slower pace than any.   At least, we should try to keep local machine and Kudu-CI the same.  Thought?

**Comment by davidebbo** [comment]
Agreed that local machine and Kudu-CI should be the same. I think the script probably works for both.

Note that this script has been pretty broken for a while, so the bar to make changes to it is not very high!

File: Setup/KuduSetupCustomWebPiFeed.xml [hunk 1]
```diff
@@ -212,4 +208,123 @@
       </installer>
     </installers>
   </entry>
-</feed>
\ No newline at end of file
+  <!-- entry for iisnode locked to a known version -->
+  <entry>
+    <productId>iisnode_0_2_11</productId>
```
Comment (davidebbo) [inline Setup/KuduSetupCustomWebPiFeed.xml]: So the reason for this is that the standard WebPI feed doesn't have those newer entries?

Comment (shrimpy) [inline Setup/KuduSetupCustomWebPiFeed.xml]: correct. it has the old version from 2012, and the old version is depends on Node.js 0.8

**Pull request merged**